File: AppMovies/app/src/main/java/com/svmdev/appmovies/data/model/Cast.java
package com.svmdev.appmovies.data.model;

import com.google.gson.annotations.SerializedName;
import com.svmdev.appmovies.data.webservice.URLs;

public class Cast {

    @SerializedName("character")
    private String character;

    @SerializedName("name")
    private String name;

    @SerializedName("profile_path")
    private String profilePath;

    @SerializedName("gender")
    private int gender;

    public Cast(String profilePath) {
        this.profilePath = profilePath;
    }

    public String getCharacter() {
        return character;
    }

    public String getName() {
        return name;
    }

    public String getProfilePath() {
        if (profilePath == null){
            return "";
        }
        String replace = profilePath.replace("'\'","");
        return URLs.imageUrl + replace;
    }

    public int getGender() {
        return gender;
    }
}

File: app/src/main/java/com/thiagoseiji/movieapp/data/Cast.kt
package com.thiagoseiji.movieapp.data

import com.google.gson.annotations.SerializedName
import io.objectbox.annotation.Id
import java.util.*


data class Cast(
    @Id(assignable = true) val id: Int,
    @SerializedName("name") val name: String,
    @SerializedName("character") val character: String,
    @SerializedName("profile_path") val profileImage: String
)